### DISCUSSION TITLE
Intentionally crash CefSharpBrowserSubprocess.exe to test crash dump handling

### DISCUSSION THREAD

**Discussion by snowball58** [cefsharp/CefSharp#3927] [Q&A] [answered]
Is there any known convenient way to intentionally crash CefSharpBrowserSubprocess.exe for testing purposes?

**Comment by amaitland** [comment, marked as answer]
> Load one of the following URLs in the CEF-based application to cause a crash:
>  - Main (browser) process crash: chrome://inducebrowsercrashforrealz
> - Renderer process crash: chrome://crash
> - GPU process crash: chrome://gpucrash

As per https://bitbucket.org/chromiumembedded/cef/wiki/CrashReporting.md#markdown-header-testing
